### PR TITLE
Feature/koning/vhost no leading slash

### PR DESCRIPTION
**Pull request by koning**
Review leading / from default vhost when using rabbitmq server.

### CODE REVIEW

File: merlin/config/broker.py [hunk 1]
```diff
@@ -213,16 +219,19 @@ def get_connection_string(include_password=True):
     if broker not in BROKERS:
         raise ValueError(f"Error: {broker} is not a supported broker.")
 
-    if broker == "rabbitmq":
+    if broker == "rabbitmq" or broker == "amqps":
         return get_rabbit_connection(config_path, include_password)
```
Comment (lucpeterson) [inline merlin/config/broker.py:223]: can we be more explicit here and add the conn="amqps" keyword?

Comment (koning) [reply merlin/config/broker.py:223]: That is is the default, as it was before.

Comment (lucpeterson) [reply merlin/config/broker.py:223]: I understand it's the default: it's mostly for self-documenting code. not important.

Comment (koning) [reply merlin/config/broker.py]: Added.

**Pull request merged**